stock_picking_cancel_reason/views/stock_picking_views.xml

```diff
@@ -0,0 +1,98 @@
+        <field name="inherit_id" ref="stock.view_picking_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//button[@name='action_cancel']" position="attributes">
+                <attribute name="invisible">True</attribute>
+            </xpath>
+            <xpath expr="//button[@name='action_cancel']" position="before">
+                <button
+                    name="%(action_stock_picking_cancel)d"
+                    states="assigned,confirmed,partially_available,draft,waiting)"
+                    string="Cancel"
+                    type="action"
+                    groups="base.group_user"
+                />
+            </xpath>
+            <xpath expr="/form/sheet/div/h1" position="after">
+                <div attrs="{'invisible': [('state', '!=', 'cancel')]}">
+                    <label for="cancel_reason_id" string="Cancellation reason:" />
+                    <field
+                        name="cancel_reason_id"
+                        class="oe_inline"
+                        options='{"no_open": True}'
+                    />
+                    <div>
+                        <label for="description" class="oe_inline" />
+                    </div>
+                    <div>
+                        <field name="description" class="oe_inline" />
+                    </div>
+                </div>
+            </xpath>
+        </field>
+    </record>
+
+    <record id="view_stock_picking_cancel_reason_form" model="ir.ui.view">
+        <field name="name">Stock Picking Cancel Reason</field>
+        <field name="model">stock.picking.cancel.reason</field>
+        <field name="arch" type="xml">
+            <form string="Stock Picking Cancel Reasons">
+                <sheet>
+                    <widget
+                        name="web_ribbon"
+                        title="Archived"
+                        bg_color="bg-danger"
+                        attrs="{'invisible': [('active', '=', True)]}"
+                    />
+                    <field name="active" invisible="True" />
+                    <group>
+                        <field name="name" />
+                    </group>
+                </sheet>
+            </form>
+        </field>
+    </record>
+
+    <record id="view_stock_picking_cancel_reason_tree" model="ir.ui.view">
+        <field name="name">Stock Picking Cancel Reason</field>
+        <field name="model">stock.picking.cancel.reason</field>
+        <field name="arch" type="xml">
+            <tree string="Stock Picking Cancel Reasons">
+                <field name="name" />
+            </tree>
+        </field>
+    </record>
+
+    <record id="view_stock_picking_cancel_reason_search" model="ir.ui.view">
+        <field name="name">Stock Picking Cancel Reason</field>
+        <field name="model">stock.picking.cancel.reason</field>
+        <field name="arch" type="xml">
+            <search string="Stock Picking Cancel Reasons">
```

Comment:
@PrapassornS Could you fix this :

![image](https://user-images.githubusercontent.com/19529533/114347260-f686db00-9b64-11eb-910f-a49ce20b9f4a.png)

Reply:
Fixed.